src/core/geometry/path/create.js

```diff
@@ -0,0 +1,19 @@
+const mat4 = require('../../math/mat4')
+
+/**
+ * Produces an empty, open path.
+ * @returns {path2} - the empty, open path.
+ * @example
+ * create()
+ */
+const create = () => {
+  return {
+    basePoints: [],  // Contains canonical, untransformed points.
+    points: [],  // Contains canonical, transformed points.
+    isClosed: false,
+    isCanonicalized: true, // Has no non-canonical points.
```

Comment:
It seems that if points is initially undefined, then we can get rid of isCanonalized and check instead for if points === undefined

Reply:
Yes, that's correct.

But I suspect it would make the code less readable, and make debugging harder.

Reply:
Ok. Let’s keep it then.